src/schema/v2/me/__tests__/index.test.js

```diff
@@ -104,4 +104,30 @@ describe("me/index", () => {
       })
     })
   })
+
+  describe("unreadNotificationsCount", () => {
+    const countQuery = gql`
+      query {
+        me {
+          unreadNotificationsCount
+        }
+      }
+    `
+
+    it("returns the number of unread notifications", () => {
+      return runAuthenticatedQuery(countQuery, {
+        notificationsFeedLoader: () => Promise.resolve({ total_unread: 12 }),
+      }).then(data => {
+        expect(data).toEqual({ me: { unreadNotificationsCount: 12 } })
+      })
+    })
+
+    it("handles a null from gravity", () => {
```

Comment:
Kind of an odd test? I would expect this integer field in the MP schema to [possibly] be non-null, and return a 0 in those cases.

Reply:
fair!